lib/profile/view/ProfileScreen.dart

```diff
@@ -30,24 +32,51 @@ class ProfileScreen extends StatelessWidget {
             return Stack(
               children: <Widget>[
                 Container(
-                  padding: EdgeInsets.symmetric(vertical: 40),
+                  padding: EdgeInsets.symmetric(
+                    vertical: mediaQuery.size.height > 400 ? 40 : 0,
+                  ),
                   alignment: Alignment.topCenter,
                   child: Column(
                     children: <Widget>[
                       CircleAvatar(
-                        //backgroundImage: NetworkImage('url'),
-                        radius: 60,
+                        backgroundImage:
+                            AssetImage('assets/profile_screen/profile_pic.png'),
+                        radius: 50,
+                        backgroundColor: Colors.transparent,
                       ),
+                      // ),
                       Container(
-                        padding: EdgeInsets.all(16),
+                        padding: EdgeInsets.all(12),
                         child: ListTile(
-                          trailing: IconButton(
-                              icon: Icon(Icons.save), onPressed: null),
+                          // trailing: IconButton(
+                          // icon: Image.asset(
+                          //     'assets/profile_screen/edit_name.png'),
+                          // onPressed: null),
                           title: TextFormField(
                             controller: nameController,
                             decoration: InputDecoration(
-                              icon: Icon(Icons.person),
-                              labelText: 'Name ',
+                              // icon: Icon(Icons.person),
+                              // labelText: 'Name ',
+
+                              fillColor: Colors.white,
+                              focusColor: Colors.white,
+                              hintText: 'Full Name',
+                              hintStyle: TextStyle(
+                                fontFamily: 'Verdana',
+                                fontSize: 16,
+                              ),
+                              suffixIcon: Image.asset(
+                                  'assets/profile_screen/edit_name.png'),
```

Comment:
The resolution of this image is quite poor, instead of exporting the asset as a png from XD, use svg for better scalable images.

Reply:
Hey,
I have updated the assets to SVG format.